scripts/setup-cfengine-build-host.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+shopt -s expand_aliases
+# install needed packages and software for a build host
+set -e
+
+if command -v wget; then
+  alias urlget=wget
+elif command -v curl; then
+  alias urlget='curl -O'
+else
+  echo "Error: need something to fetch URLs. Didn't find either wget or curl."
+  exit 1
+fi
+if grep -i suse /etc/os-release; then
+  # todo check version of suse, 12 or 15
+  urlget https://cfengine-package-repos.s3.amazonaws.com/enterprise/Enterprise-3.21.4/agent/agent_suse15_x86_64/cfengine-nova-3.21.4-1.suse15.x86_64.rpm
+  sudo zypper install -y cfengine-nova-3.21.4-1.suse15.x86_64.rpm
+else
+  urlget https://s3.amazonaws.com/cfengine.packages/quick-install-cfengine-enterprise.sh
+  echo "c358ca0e0dce49e8784ff2352e7c94356332ded80f5ca3903b0b3dc8d6a10cf4  quick-install-cfengine-enterprise.sh" | sha256sum --check -
+  chmod +x quick-install-cfengine-enterprise.sh
+  sudo bash ./quick-install-cfengine-enterprise.sh agent
+fi
+
+# get masterfiles
+urlget https://cfengine-package-repos.s3.amazonaws.com/enterprise/Enterprise-3.21.4/misc/cfengine-masterfiles-3.21.4-1.pkg.tar.gz
+
+echo "a4b35ad85ec14dda49b93c1c91a93e09f4336d9ee88cd6a3b27d323c90a279ca	cfengine-masterfiles-3.21.4-1.pkg.tar.gz" | sha256sum --check -
+
+tar xf cfengine-masterfiles-3.21.4-1.pkg.tar.gz
+sudo cp -a masterfiles/* /var/cfengine/inputs/
+
+urlget https://raw.githubusercontent.com/craigcomstock/cfengine-security-hardening/e3039050296ec20c7e44b3accba84c146cf6ef69/upgrade-all-packages/upgrade_all_packages.cf
```

Comment:
Perhaps better to fetch from cfengine build infra rather than your personal repo?

Reply:
will research this. But this is the "source" for the build module but yes agreed if possible I will change that.